Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -9829,6 +9829,25 @@ a_Player:OpenWindow(Window);
 					},
 					Notes = "Removes a currently applied entity effect",
 				},
+				TeleportToCoords =
```

Comment:
I have two questions/concerns:
- How do the API docs handle this override? Do we have other cases where we do this?
- Can this export ever be reverted in the future if necessary, without breaking things?

It's been a while since I touched any of this. If everything is fine, feel free to dismiss this comment.

Reply:
I honestly don't know how this api export works ^^ I though it would be better to use inheritence in this patch but it's indeed not strictly necessary so I can just do a dirty patch instead ^^